fs/shm/shm_open.c

```diff
@@ -165,7 +166,7 @@ int shm_open(FAR const char *name, int oflag, mode_t mode)
   if (ret < 0)
     {
       set_errno(-ret);
-      return -1;
+      return ERROR;
```

Comment:
This is matter of opinion of course,but I find these changes confusing. Posix spec says that the function is supposed to return -1. Not "ERROR", which is nuttx internal define.